docker/Dockerfile

```diff
@@ -54,5 +64,8 @@ RUN bash -c "source /opt/build.sh; GetNrfSdk;"
 # McuBoot
 RUN bash -c "source /opt/build.sh; GetMcuBoot;"
 
+# If we are a devcontainer add the infinitime user
+RUN if [ -n "$DEVCONTAINER" ] ; then adduser infinitime ; fi
+    
```

Comment:
What's this used for in the devcontainer?

Reply:
After the devcontainer is built, VSCode effectively remotes into it. It is convention to not use the root user for this, but instead create another user.